store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9479,11 +9479,11 @@ TODO: delete them permanently from here
   <desc>Expiry time for password reset recovery code</desc>
 </attr>
 
-<attr id="2139" name="zimbraRecoveryEmailVerificationData" type="string" cardinality="single" optionalIn="account" since="8.8.9">
+<attr id="2139" name="zimbraRecoveryAccountVerificationData" type="string" cardinality="single" optionalIn="account" since="8.8.9">
   <desc>Recovery email verification data</desc>
 </attr>
 
-<attr id="2140" name="zimbraRecoveryEmailCodeValidity" type="duration" cardinality="single" optionalIn="account,cos" flags="accountInfo,accountInherited" since="8.8.9">
+<attr id="2140" name="zimbraRecoveryAccountCodeValidity" type="duration" cardinality="single" optionalIn="account,cos" flags="accountInfo,accountInherited" since="8.8.9">
```

Comment:
I'm guessing we have not shipped any code which uses these attributes (2139 & 2140), otherwise you would not be renaming them, correct?

Reply:
yes, these attributes are not merged in develop.